### ISSUE TITLE
"poetry run pytest" on main branch returns 14 Import Errors

### ISSUE THREAD

**Issue by MoizKhuzema** [embedchain/embedchain#1403]
### 🐛 Describe the bug

@Dev-Khant  When I run tests using the command "poetry run pytests", it returns 14 import errors and module not found errors. 
Here is an example of 3 errors:
<img width="659" alt="errors" src="https://github.com/embedchain/embedchain/assets/87359666/a1ee8100-f30c-4deb-b679-81fd4ab24aac">

I believe this is happening because Poetry is failing to install many dependencies when running "poetry install --all-extras" because the hash of the downloaded file doesn't match the expected hash recorded in the lock file.

**Comment by Dev-Khant** [comment]
In order to lighten the package we are removing few packages from poetry and so for those packages manual installation is required.

Here you can delete you current `poetry.lock` and then do `poetry install --all-extras` so most of the packages you got error for will be installed and rest you can installed manually.

**Comment by Dev-Khant** [comment]
@MoizKhuzema Let me know if this works or not so I can close this issue. 
Anyways I'm going to update the contributing docs.

**Comment by MoizKhuzema** [comment]
> @MoizKhuzema Let me know if this works or not so I can close this issue. Anyways I'm going to update the contributing docs.

@Dev-Khant I managed to resolve all import and module errors successfully. I had to pip install all the modules that were not installed by poetry and it was a lot of work. I have a list of all the libraries I installed manually and the versions I used. I believe that can be helpful for anyone else trying to run the project for the first time. 

here is the list:
pip install --upgrade 'embedchain[milvus]' 
pip install --upgrade 'embedchain[weaviate]'
pip install qdrant-client==1.9.1
pip install pinecone-text pinecone-client
pip install --upgrade "embedchain[elasticsearch]"
pip install youtube-transcript-api=0.6.1
pip install psycopg==3.1.19
pip install "embedchain[googledrive]"
pip install --upgrade "embedchain[gmail]"
pip install "unstructured[local-inference, all-docs]"
pip install dropbox==11.36.2
pip install langchain-google-vertexai
pip install ollama
pip install google-generativeai==0.3.0
pip install --upgrade "embedchain[opensearch]"
pip install slack-sdk==3.21.3
pip install mysql-connector-python==8.1.0
pip install gpt4all==2.0.2
pip install PyGithub gitpython
pip install --upgrade "embedchain[together]"
pip install --upgrade "embedchain[llama2]" 
pip install sentence-transformers 
pip install langchain-anthropic 

it is important that langchain-anthropic is installed at the end or it will create more import errors

<img width="492" alt="pic1" src="https://github.com/embedchain/embedchain/assets/87359666/03718b63-0f22-43b5-8916-58a9b516c2e6">
